.github/workflows/build.yml

```
@@ -0,0 +1,57 @@
+name: Build images
```

```suggestion
name: docker
```
We use kebab case for all our ci names